packer/grapl-service/files/consul.conf.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Note: All the Packer/Consul stuff in here, I can't quite speak to it being correct or working yet - it's primarily just stuff I inherited from Colin. The idea is:
- get the AMI CI in
- iterate on it from there